sql/moz-fx-data-marketing-prod/iprospect/adspend_meta_v1/query.sql

```diff
@@ -0,0 +1,58 @@
+SELECT
+  @submission_date AS submission_date,
+  COUNT(DISTINCT today.date) = 30 AS has_30_days_of_data,
+  COUNT(DISTINCT today.date) = 0 AS has_missing_data,
+  COUNTIF(yesterday.fetch_ad_name IS NULL AND yesterday.date < @submission_date) AS new_row_count,
+  COUNTIF(
+    today.client_cost != yesterday.client_cost
+    AND yesterday.client_cost IS NOT NULL
+    AND today.client_cost > 0
+  ) AS client_cost_nonzero_changed_count,
+  COUNTIF(
+    today.imps_vendor != yesterday.imps_vendor
+    AND yesterday.imps_vendor IS NOT NULL
```

Comment:
What is imps?

Reply:
vendor impressions. I figured it might be best to just keep the original naming for these to avoid any unexpected naming mismatches.